pages/spicedb/api/_meta.json

```diff
@@ -5,7 +5,10 @@
     "newWindow": true
   },
   "http-api": {
-    "title": "HTTP API Reference",
+    "title": "HTTP API Reference"
+  },
+  "postman": {
+    "title": "Postman Collection",
     "href": "https://www.postman.com/authzed/workspace/spicedb/overview",
     "newWindow": true
   }
```

Comment:
The Postman collection only includes HTTP stuff; perhaps it could be a child of the HTTP API, but I'm also OK as it is